Library/Homebrew/cask/artifact/artifact.rb

```diff
@@ -33,9 +33,6 @@ def resolve_target(target)
       end
 
       sig { params(cask: Cask, source: T.any(String, Pathname), target: T.any(String, Pathname)).void }
-      def initialize(cask, source, target:)
-        super(cask, source, target: target)
-      end
```

Comment:
Sig needs removing here too

Reply:
As long as `super` has the same signature, it should be fine.

Reply:
yeah misread the super thing initially - sig should still be removed though